try {
    abstract.a;
}
catch (error) {
    console.log("CATCH ERROR:", error);
}
finally {
    console.log("final");
}
